Resources/ProxyProvider/DIRACCAProxyProvider.py

```diff
@@ -3,18 +3,13 @@
     This class is a simple, limited CA, its main purpose is to generate a simple proxy for DIRAC users
     who do not have any certificate register on the fly.
 
-    Required parameters in the DIRAC configuration for its implementation::
-
-    <Provider Name> section:
-
-    *  ProviderType = DIRACCA,
-    *  CertFile = <CA sertificate path>,
-    *  KeyFile = <CA key path>,
-    *  Match = <Match DNs>,  # For ex.: 'Match = O, OU'
-    *  Supplied = <Supplied DNs>,
-    *  Optional = <Optional DNs>,
-    *  DNOrder = <Preferred DNs order as list>,  # For ex.: 'DNOrder = O, OU, CN, emailAddress'
-    *  <Some distinguished name type>: <Default value>,  # For ex.: 'OU = CA'
+    Required parameters in the DIRAC configuration for its implementation:
+
+    .. literalinclude:: /dirac.cfg
```

Comment:
This is not the correct location, I believe...? But @andresailer should confirm.

Reply:
`dirac.cfg` is copied in `docs\source\` during docs building and deleted during cleaning. This is done to avoid `../../../dirac.cfg` like paths. Above is a discussion on this.

Reply:
Yes, this `/dirac.cfg` is the right thing to do here.